Cargo.toml

```diff
@@ -18,17 +18,19 @@ members = ["magnus-macros"]
 exclude = ["examples/rust_blank/ext/rust_blank", "examples/custom_exception_ruby/ext/ahriman", "examples/custom_exception_rust/ext/ahriman"]
 
 [features]
+default = []
 embed = ["rb-sys/link-ruby"]
 rb-sys-interop = []
 ruby-static = ["rb-sys/ruby-static"]
 
 [dependencies]
 magnus-macros = { version = "0.1.0", path = "magnus-macros" }
-rb-sys = { version = "~0.9.34", default-features = false, features = ["bindgen-rbimpls", "bindgen-deprecated-types"] }
+rb-sys = { version = "~0.9.37", default-features = false, features = ["bindgen-rbimpls", "bindgen-deprecated-types"] }
 
 [dev-dependencies]
-magnus = { path = ".", features = ["embed", "rb-sys-interop"] }
+magnus = { path = ".", features = ["embed", "rb-sys-interop"], default-features = false }
 
 [package.metadata.docs.rs]
-all-features = true
+no-default-features = true
+features = ["embed", "rb-sys-interop"]
```

Comment:
While this isn't a required diff, I think making the features explicit here makes it clear that we need the `embed` feature for docs to generate.